kubetest/main.go

```diff
@@ -657,6 +682,9 @@ func prepare(o *options) error {
 	}); err != nil {
 		return err
 	}
+	if o.parallelism < 0 {
+		o.parallelism = defaultParallelism
+	}
```

Comment:
Er, I see what you're asking now. Sorry, that crept in there after I moved it. :)